On prem installation revamp (#2164)
mendersoftware/mender-docs: 07.Server-installation/01.Overview/01.Architecture/docs.md

---
title: Architecture
taxonomy:
    category: docs
---

> !!! Enterprise server will have an enterprise version of the container pulled which will have enterprise features.

Reviewer: ```suggestion
!!! The Enterprise server runs different versions of the containers providing the enterprise features.
```

> !!! For evaluation of the Mender Enterprise Server please [contact us](https://mender.io/contact-us) to gain access to the enterprise containers.

Reviewer: ```suggestion
!!! For evaluation of the Mender Enterprise Server, please [contact us](https://mender.io/contact-us) to gain access to the enterprise containers.
```

> The architecture below show the case for the Open Source Mender Server that gets set up with the docker compose evaluation.

Reviewer: ```suggestion
The architecture below shows the Open Source Mender Server case set up with the docker-compose evaluation.
```

- [Mender Device Configuration Service](https://github.com/mendersoftware/deviceconfig?target=_blank)
- [Mender Device Connect Service](https://github.com/mendersoftware/deviceconnect?target=_blank)
- [Mender Device Monitor Service](https://github.com/mendersoftware/devicemonitor?target=_blank)
- [Mender Deployments Service](https://github.com/mendersoftware/deployments?target=_blank)
- [Mender Device Inventory Service](https://github.com/mendersoftware/inventory?target=_blank)
- [Mender User Administration Service](https://github.com/mendersoftware/useradm?target=_blank)
- [Mender Workflows Service](https://github.com/mendersoftware/workflows?target=_blank)
- [Mender Reporting Service](https://github.com/mendersoftware/reporting?target=_blank)
- [Mender Create Artifact Worker](https://github.com/mendersoftware/create-artifact-worker?target=_blank)
- API Gateway based on [Traefik](https://doc.traefik.io/traefik/?target=_blank)
- [Minio](https://www.minio.io?target=_blank) object storage
- [NATS.io](https://nats.io?target=_blank) messaging system
- [OpenSearch](https://www.opensearch.org?target=_blank) storage and search engine

> Services are delivered in form of Docker images, available from

Reviewer: ```suggestion
Services are delivered as Docker images, available from the
```

official [Mender Docker repository](https://hub.docker.com/r/mendersoftware/?target=_blank).

> When required, each service can be built directly from its source code. Consult

Reviewer: ```suggestion
When required, each service can be built directly from its source code. Consult the
```
